listenbrainz/listenstore/tests/test_timescalelistenstore.py

```diff
@@ -72,6 +73,27 @@ def _create_test_data(self, user_name, test_data_file_name=None):
         self.logstore.insert(test_data)
         return len(test_data)
 
+    def _insert_with_created(self, listens):
+        """
+            Insert a batch of listens. Returns a list of (listened_at, track_name, user_name) that indicates
```

Comment:
This comment isn't right since this function does not return any data.

Reply:
Fixed it.